src/features/presentation_mode/LegendsPanel/LegendsPanel.tsx

```
@@ -0,0 +1,9 @@
+import { PanelContent } from '~features/legend_panel/components/PanelContent/PanelContent';
+
+export function LegendsPanel() {
+  return (
+    <div className="knt-panel">
+      <PanelContent></PanelContent>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)